### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Bupilipili**
In this branch.

-Added MIT.md file to the repository
-Updated changes in the README.md file
-Updated the style.css file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,7 +147,8 @@ Example command:
   rails server
 ```
 --->
-
+   sh
+   rails server
```
Comment (bobb-Rob) [inline README.md:151]: - [ ] To run this project, the `sh rails server` you have in your README is not helpful to new users of your repository. Kindly use something that works. E.g. Open the index.html file on the browser or use a vs code extension `server` to start the project`

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Bupilipili, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,41 +1,14 @@
...
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```
Comment (Qoosim) [inline README.md:6]: - [OPTIONAL]  For your subsequent projects, ensure you remove the logo on the template used and replace it with your logo if you have any.

logo

File: README.md [hunk 3]
```diff
@@ -241,30 +124,21 @@ Feel free to check the [issues page](../../issues/).
...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
 - **[Question_1]**
```
Comment (Qoosim) [inline README.md:144]: - [OPTIONAL] It is optional to remove this part if there's nothing to put there.

fqa

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Bupilipili, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**